### PR TITLE
Add multi-cluster writes s3 blog

### PR DESCRIPTION
**Pull request by dennyglee**
Add the multi-cluster writes S3 blog

### CODE REVIEW

File: src/blog/2022-05-18-multi-cluster-writes-to-delta-lake-storage-in-s3/index.mdx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+##### 2. Follow the configuration steps listed in [single-cluster setup](https://docs.delta.io/1.2.0/delta-storage.html#-delta-storage-s3-single-cluster)’s Configuration section.
+
+##### 3. Include the JARs for `delta-storage-s3-dynamodb` and `aws-java-sdk` in the classpath.
+
+##### 4. Configure the LogStore implementation in your Spark session.
+
+First, configure this LogStore implementation for the scheme s3.
+
+```scala
+spark.delta.logstore.s3.impl=io.delta.storage.S3DynamoDBLogStore
```
Comment (scottsand-db) [inline src/blog/2022-05-18-multi-cluster-writes-to-delta-lake-storage-in-s3/index.mdx:87]:
```
spark.delta.logStore.s3.impl=io.delta.storage.S3DynamoDBLogStore
```

`logStore` !!! (super important :) )

File: src/blog/2022-05-18-multi-cluster-writes-to-delta-lake-storage-in-s3/index.mdx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+```
+
+Next, specify additional information necessary to instantiate the DynamoDB client. You must instantiate the DynamoDB client with the same tableName and region each Spark session for this multi-cluster mode to work correctly. A list of per-session configurations and their defaults is given below:
+
+| Configuration Key                                               | Description                                     | Default                                                                                                                                                  |
+| --------------------------------------------------------------- | ----------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `io.delta.storage.S3DynamoDBLogStore.ddb.tableName`             | The name of the DynamoDB table to use           | `delta_log`                                                                                                                                              |
```
Comment (scottsand-db) [inline src/blog/2022-05-18-multi-cluster-writes-to-delta-lake-storage-in-s3/index.mdx:94]: `spark.io.delta....`

starts with `spark`

File: src/blog/2022-05-18-multi-cluster-writes-to-delta-lake-storage-in-s3/index.mdx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+| `io.delta.storage.S3DynamoDBLogStore.ddb.region`                | The region to be used by the client             | `us-east-1`                                                                                                                                              |
+| `io.delta.storage.S3DynamoDBLogStore.credentials.provider`      | The AWSCredentialsProvider\* used by the client | [`DefaultAWSCredentialsProviderChain`](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/auth/DefaultAWSCredentialsProviderChain.html) |
```
Comment (scottsand-db) [inline src/blog/2022-05-18-multi-cluster-writes-to-delta-lake-storage-in-s3/index.mdx:96]: spark.io.delta.storage.S3DynamoDBLogStore.credentials.provider

File: src/blog/2022-05-18-multi-cluster-writes-to-delta-lake-storage-in-s3/index.mdx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+| `io.delta.storage.S3DynamoDBLogStore.provisionedThroughput.rcu` | (Table-creation-only\*\*) Read Capacity Units   | `5`                                                                                                                                                      |
```
Comment (scottsand-db) [inline src/blog/2022-05-18-multi-cluster-writes-to-delta-lake-storage-in-s3/index.mdx:97]: spark.io.delta.storage.S3DynamoDBLogStore.provisionedThroughput.rcu

File: src/blog/2022-05-18-multi-cluster-writes-to-delta-lake-storage-in-s3/index.mdx [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+| `io.delta.storage.S3DynamoDBLogStore.provisionedThroughput.wcu` | (Table-creation-only\*\*) Write Capacity Units  | `5`                                                                                                                                                      |
```
Comment (scottsand-db) [inline src/blog/2022-05-18-multi-cluster-writes-to-delta-lake-storage-in-s3/index.mdx:98]: spark.io.delta.storage.S3DynamoDBLogStore.provisionedThroughput.wcu

File: src/blog/2022-05-18-multi-cluster-writes-to-delta-lake-storage-in-s3/index.mdx [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+Ensure that all nodes and/or Spark drivers in your multi-node cluster use this LogStore implementation as well as the same DynamoDB table and region. If some drivers use out-of-the-box Delta Lake while others use this experimental LogStore, there is the potential for data loss.
+
+### Requirements
+
+- S3 credentials: [IAM roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html) (recommended) or access keys
+- DynamoDB operating permissions
+- Apache Spark associated with the corresponding Delta Lake version.
+- Hadoop’s [AWS connector (hadoop-aws)](https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-aws/) for the version of Hadoop that Apache Spark is compiled for.
+- JAR Files: <code>delta-storage-s3-dynamodb artifact</code>
```
[the remark this answers is not in the document]
Comment (scottsand-db) [reply src/blog/2022-05-18-multi-cluster-writes-to-delta-lake-storage-in-s3/index.mdx:51]: include the link

File: src/blog/2022-05-18-multi-cluster-writes-to-delta-lake-storage-in-s3/index.mdx [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+### Setup Configuration
+
+##### 1. Create the DynamoDB table.
```
[the remark this answers is not in the document]
Comment (dennyglee) [reply src/blog/2022-05-18-multi-cluster-writes-to-delta-lake-storage-in-s3/index.mdx:55]: Too big with 4 `#` which is why I went for 5 `#`

**Review by scottsand-db — APPROVED** [review]

**Pull request merged**